src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_metadata.py

```diff
@@ -518,13 +522,14 @@ def test_data_dont_fit_in_mdblob(metadata_store):
             title='test' + str(x),
             infohash=random_infohash(rng),
             id_=rng.randint(0, 100000000),
+            torrent_date=int2time(rng.randint(0, 4000000)),
             timestamp=rng.randint(0, 100000000),
         )
         for x in range(0, 1)
     ]
     chunk, index = entries_to_chunk(md_list, chunk_size=1)
     assert index == 1
-    assert len(chunk) == 206
+    assert len(chunk) == 205
```

Comment:
what is reason behind length reduction by 1?

Reply:
I've set a fixed random seed for `torrent_date`, so the serialized payload becomes now completely deterministic. Well, the new content seem to compress a little bit better, so is loses a single byte :wink: